core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1386,6 +1387,14 @@ public Void visitTableExecute(TableExecuteNode node, Void context)
             return processChildren(node, context);
         }
 
+        @Override
+        public Void visitTableExecuteCoordinatorOnly(TableCoordinatorExecuteNode node, Void context)
+        {
+            addNode(node, "ExecuteCoordinatorOnly", format("[%s]", node.getTarget()));
```

Comment:
node.getTarget does not have toStriong - but I suggest to drop the class altogether